### ISSUE TITLE
TypeInContainerPlugin incorrectly breaks block when there is inline-block element with DIV inside

### ISSUE THREAD

**Issue by JiuqingSong** [microsoft/roosterjs#619]
```html
<div>line1<span style="display:inline-block"><div>line1.1</div></span></div>
```

Let editor load the HTML code above, TypeInContainerPlugin will break it into two lines.

**Comment by JiuqingSong** [comment]
need content state suppoert

**Comment by JiuqingSong** [comment]
This will be solved in #1553
